chart/values.yaml

```diff
@@ -72,23 +72,29 @@ ingress:
   className: ""
 
   # -- Ingress annotations
-  annotations: { }
-  # nginx.ingress.kubernetes.io/rewrite-target: /$2
-  # nginx.ingress.kubernetes.io/use-regex: "true"
-  # nginx.ingress.kubernetes.io/ssl-redirect: 'true'
+  annotations:
+    # -- Specifies number of requests accepted from a given IP each minute
+    nginx.ingress.kubernetes.io/limit-rpm: "180"
```

Comment:
could you leave this empty y default?
You can keep the rpm commented out.

Reply:
addressed, thanks